tests/modeladmin/tests.py

```diff
@@ -615,6 +617,49 @@ def test_log_actions(self):
         self.assertEqual(ma.log_change(mock_request, self.band, 'changed'), LogEntry.objects.latest('id'))
         self.assertEqual(ma.log_change(mock_request, self.band, 'deleted'), LogEntry.objects.latest('id'))
 
+    @mock.patch('django.contrib.admin.options.get_deleted_objects')
```

Comment:
These tests aren't what's needed. The implementation details of `ModelAdmin.get_deleted_objects()` aren't important. What the comment in the ticket is saying is that a test for the change in `django/contrib/admin/actions.py`, for example, should be tested. Right now, that change can be reverted and no tests are failing. bf91be83d56d70181de807a5e8d22679cf63a52f is a similar hook that might give you an idea of what's required.